lib/options/index.ts

```diff
@@ -77,6 +77,7 @@ export function buildOptions(args: OktaAuthOptions = {}): OktaAuthOptions {
     codeChallengeMethod: args.codeChallengeMethod,
     recoveryToken: args.recoveryToken,
     activationToken: args.activationToken,
+    useGenericRemediator: args.useGenericRemediator,
```

Comment:
Do you think we should have an `idx` section to the configuration for these type of options?

```
const authClient = new OktaAuth({
  issuer,
  clientId,
  idx: {
    useGenericRemediator: true,
    exchangeCodeForTokens: false
  }
});
```

Essentially preloading options which would be passed to `idx.run` or `idx.proceed`

Reply:
How about we still leave/add (not sure if it exists) `exchangeCodeForTokens` as a SDK level option? I feel it applies to both interaction and authorization code flows.

Reply:
It does not apply to authorization code flow - there is no automatic token retrieval in the `tx` pipeline. It will always end with a redirect (or post message response from authorize page)

Reply:
I am not sure if `authn` is an extension of oidc, but this config can control if redirect endup with a code or tokens.